src/libical-glib/i-cal-object.c.in

```diff
@@ -586,8 +586,8 @@ void i_cal_object_set_owner(ICalObject *iobject, GObject *owner)
  * if not NULL, is referenced for thread safety. Unref it with g_object_unref
  * when done with it.
  *
- * Returns: (transfer full) (allow-none): Current owner of the libical
- *    native structure. returns NULL, when there is no owner.
+ * Returns: (transfer full) (nullable): Current owner of the libical
+ *    native structure. returns %NULL, when there is no owner.
```

Comment:
Bonus points, if you make it `Returns %NULL....` (a capital `R` after a dot).

Reply:
applied

Reply:
Thanks.